examples/bzlmod/cross_compile/README.md

```diff
@@ -1,6 +1,290 @@
-# bzlmod cross-compile example
+# Cross Compilation
 
-This example shows how to use `rules_rust` through bzlmod to invoke Rust cross-compilation.
+For cross compilation, you have to specify a custom platform to let Bazel know that you are compiling for a different platform than the default host platform.
 
-It should be possible to `bazel build //:hello_world_aarch64` and `bazel build //:hello_world_x86_64` regardless of your
-host platform (as long as it is supported by Bazel and rustc).
+The example code is setup to cross compile from the following hosts to the the following targets:
+
+* {linux, x86_64} -> {linux, aarch64}
+* {linux, aarch64} -> {linux, x86_64}
+* {darwin, x86_64} -> {linux, x86_64}
+* {darwin, x86_64} -> {linux, aarch64}
+* {darwin, aarch64 (Apple Silicon)} -> {linux, x86_64}
+* {darwin, aarch64 (Apple Silicon)} -> {linux, aarch64}
+
+You cross-compile by calling the target.
+
+`bazel build //:hello_world_x86_64`
+
+or
+
+`bazel build //:hello_world_aarch64`
+
+Notice, the default target `//...` does not know about the 
+many different target platform and will report an error. 
```

Comment:
I'm not sure what problem this is referring to, and this seemed to work for me - can you elaborate / give an example of the issue?

Reply:
Correct, I just checked it a moment ago. 
Seems to be a leftover from. Thanks for catching it. 

I've updated the Readme to reflect that  target `//...` works as expected.